store/sqlstore/group_store.go

```diff
@@ -312,6 +312,41 @@ func (s *SqlGroupStore) GetMemberCount(groupID string) (int64, *model.AppError)
 	return count, nil
 }
 
+func (s *SqlGroupStore) GetMemberUsersNotInChannel(groupID string, channelID string) ([]*model.User, *model.AppError) {
+	var groupMembers []*model.User
+
+	query := `
+		SELECT
+			Users.*
+		FROM
+			GroupMembers
+			JOIN Users ON Users.Id = GroupMembers.UserId
+		WHERE
+			GroupMembers.DeleteAt = 0
+			AND Users.DeleteAt = 0
```

Comment:
nit: I'd move this lines to the bottom to keep consistent to having DeleteAt checks at the end.